framework/src/org/apache/cordova/DroidGap.java

```diff
@@ -30,5 +30,6 @@ Licensed to the Apache Software Foundation (ASF) under one
  */
 @Deprecated
 public class DroidGap extends CordovaActivity {
-
+    @Override
+    public void onXWalkReady() {}
```

Comment:
This empty implementation will override CordovaActivity.onXWalkReady.

According to the doc, this seems a deprecated Activity replaced by CordovaActivity, so why we make it different here?